trin-core/src/portalnet/overlay_service.rs

```diff
@@ -1385,3 +1447,7 @@ mod tests {
         };
     }
 }
+
+fn should_store(_key: &Vec<u8>) -> bool {
+    return true;
+}
```

Comment:
If there's no github issue to implement this, could you create one?

Reply:
This is implemented in storage.rs, but storage.rs isn't implemented into the codebase yet.